PurrgentCare.postman_collection.json

```diff
@@ -677,6 +677,57 @@
 					},
 					"response": []
 				},
+				{
+					"name": "ToggleAccount",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"status test\", function () {\r",
+									"    pm.response.to.have.status(201);\r",
+									"});"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "PUT",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"active\": \"False\",\r\n    \"Date\": \"05-30-2024\"\r\n    }",
```

Comment:
Do we need the Date in the body of the request?
Does postman need to be updated?